scripts/dts/edtlib.py

```diff
@@ -1508,6 +1508,14 @@ def __init__(self, path, fname2path, raw=None,
             if key.endswith("-cells"):
                 self.specifier2cells[key[:-len("-cells")]] = val
 
+        # Make child binding compatibles match ours if they are missing.
```

Comment:
would it make sense to do this after line(s) 1492? where we initially set self.child_binding?

Reply:
> would it make sense to do this after line(s) 1492? where we initially set self.child_binding?

I think we want to do this after 1500 at least (where we call `self._check`) to make sure we're doing it on a well-formed binding. Doing it after initializing the look up tables means the parent i fully initialized before we go and fix up the child, which seems like a nice thing to have done in case we need to fix up the child look up tables based on the parent eventually, etc.

Happy to move it after self._check though if you want.

Reply:
Fine as, is just curious.